### PR TITLE
React router dom films

### PR DESCRIPTION
**Pull request by EkintoQ**

### CODE REVIEW

File: src/main/frontend/src/components/navbar/Navlist.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import './Navlist.css'
+import {Link} from "react-router-dom";
+const Navlist = () => {
+
+    return (
+        <ul className="Navlist">
+            <div className="Search">
+                <input type="text" className="Round"/>
+                <input type="submit" className="Submit" value="+"/>
```
Comment (prymakD) [inline src/main/frontend/src/components/navbar/Navlist.js:9]: Why it is "+" sign for search bar?:)

Comment (prymakD) [inline src/main/frontend/src/components/navbar/Navlist.js:9]: Why it is "+" sign for search bar?:)

**Review by prymakD — COMMENTED** [review]
Generally ok
- the white stripe beneath the main screen looks strange 
- should the main poster("Duna" one) change in the carousel?
- picture of the posters in the carousel time to time cover the bio of the movie

**Review by AntonPazniak — APPROVED** [review]
the design is good, I suggest only to change the movie poster and description in some places

**Review by prymakD — APPROVED** [review]
ok, checked

**Pull request merged**